src/main/java/com/wavefront/opentracing/WavefrontTracer.java

```diff
@@ -505,6 +509,7 @@ public Builder excludeJvmMetrics() {
      * @return {@code this}
      */
     public Builder redMetricsCustomTagKeys(Set<String> redMetricsCustomTagKeys) {
+      this.redMetricsCustomTagKeys.add(SPAN_KIND.getKey());
```

Comment:
this should probably be done in `build()` because `redMetricsCustomTagKeys()` might not get called